roles/common/tasks/serverspec.yml

```diff
@@ -0,0 +1,6 @@
+---
+- name: operating system serverspec
```

Comment:
above this is where I was expecting a dir to be specified to exist, since that's all we really care about here.